Sources/PointFreeRouter/Routes.swift

```diff
@@ -19,6 +19,7 @@ public enum Route: DerivePartialIsos, Equatable {
   case appleDeveloperMerchantIdDomainAssociation
   case blog(Blog)
   case discounts(code: Stripe.Coupon.Id, Pricing.Billing?)
+  case endGhosting
```

Comment:
this route is not in the admin routes because you will not be logged in as an admin when you need to end ghosting